make_manifest.py

```diff
@@ -0,0 +1,28 @@
+import hashlib, os, sys, argparse
+
+sep=','
```

Comment:
We're making all manifests comma delimited to unify all manifests with current GET manifest specs correct?

Reply:
correct - we don't want GET to redo anything so we are matching their spec

Reply:
yes